gnocchi/rest/api.py

```diff
@@ -1293,11 +1300,16 @@ def _ResourceSearchSchema():
                     u"<=", u"≤", u"le",
                     u">=", u"≥", u"ge",
                     u"!=", u"≠", u"ne",
-                    u"like"
                 ): voluptuous.All(
                     voluptuous.Length(min=1, max=1),
                     {"id": _ResourceUUID,
-                     six.text_type: ResourceSearchSchemaAttributeValue},
+                     NotIDKey: ResourceSearchSchemaAttributeValue},
+                ),
+                voluptuous.Any(
```

Comment:
It does not look like you need `voluptuous.Any` here if it's just to pass one value.